boot-speed/clouds/measure-cloud.py

```diff
@@ -15,13 +15,66 @@
 import os
 import paramiko
 import pycloudlib
+import shutil
 import sys
 
 from pathlib import Path
 
 
 known_clouds = ['ec2', 'gce']
 
+class EC2Instspec():
+    def __init__(self, release):
+        self.release = release
+
+    # Defaults
+    region = "us-east-1"
+    inst_type = "t2.micro"
```

Comment:
What's the rationale for using t2.micro instances?

(I know this is just a refactor, so I don't want to block on this question.)

Reply:
It just a default, chosen for two reasons:
* It's pycloudlib's default too
* It's *not* an instance type we are interested in measuring for the boot-speed KPI, the actual instance types we want are specified by the Jenkins job using the command line parameter. If a job ends up measuring t2.micro it means that I forgot to specify the instance type, or more in general that something went wrong :)